setup.cfg

```
@@ -1,14 +1,11 @@
-# pytest PEP8 configuration
+# pytest pycodestyle configuration
 [tool:pytest]
-pep8maxlinelength = 80
-pep8ignore =
-    docs/conf.py ALL
-    *.py E251
-    *.py W503
-    *.py E126
-
-# ignoring W503: line break occurred before a binary operator
-# ignoring E126: continuation line over-indented for hanging indent
+codestyle_max_line_length = 80
+codestyle_ignore =
```

can you please add descriptions for the ignored errors?